odl/space/pspace.py

```diff
@@ -154,20 +154,21 @@ class instance can be retrieved from the space by the
 
             Cannot be combined with: ``dist``
 
-        Returns
-        -------
-        prodspace : `ProductSpace`
-
         See also
         --------
         ProductSpaceVectorWeighting
         ProductSpaceConstWeighting
 
         Examples
         --------
-        >>> import odl
+        Product of r2 and r3
```

Comment:
Better: "Create the product of R^2 and R^3:"
Writing of the spaces is debatable, of course.